ocaml/xapi/xapi.ml

```diff
@@ -444,7 +447,60 @@ let attempt_pool_hello my_ip =
               [localhost_uuid] ;
             Some Permanent
         | `ok ->
-            None
+            let obj_uuid = Helpers.get_localhost_uuid () in
```

Comment:
We've already got `localhost_uuid` above, which can be reused.

Reply:
Ah, yes, sorry!